apps/vs-code-designer/src/package.json

```diff
@@ -847,7 +847,7 @@
           },
           "azureLogicAppsStandard.autoBinariesInstallation": {
             "type": "boolean",
-            "description": "Start using binaries installation",
+            "description": "Start using binaries installation Test",
```

Comment:
should be removed?

Reply:
Yep, just made the change so Esther can review the strings 😅